### PR TITLE
Mirror jest tests

### PR DESCRIPTION
**Pull request by bthaile**
remove manual mocks and bring over v1 jest tests that were first merged from v2. should have parity now.

### CODE REVIEW

File: packages/augur-ui/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (bthaile) [inline packages/augur-ui/babel.config.js]: seems we need to remove .babelrc if we bring babel.config.js.

File: packages/augur-ui/package.json [hunk 2]
```diff
@@ -101,27 +101,34 @@
...
+    "@types/jest-environment-puppeteer": "^2.2.1",
+    "@types/puppeteer": "^1.6.4",
+    "autoprefixer": "9.1.5",
+    "babel-core": "7.0.0-bridge.0",
     "babel-eslint": "9.0.0",
-    "babel-plugin-istanbul": "4.1.6",
-    "babel-plugin-transform-react-remove-prop-types": "0.4.20",
-    "chai": "4.2.0",
-    "chai-subset": "1.6.0",
+    "@babel/core": "7.2.0",
```
Comment (justinbarry) [inline packages/augur-ui/package.json]: These have all been hoisted. I imagine this file shouldn't change much save for the script section

**Review by justinbarry — APPROVED** [review]
LGTM

**Comment by bthaile** [comment]
Thanks, I’ll pull the branch down and some reviewing and running.

**Pull request merged**